core/src/test/scala/integration/kafka/api/AdminClientTest.scala

```diff
@@ -106,7 +106,7 @@ class AdminClientTest extends IntegrationTestHarness with Logging {
 
     val consumerSummaries = client.describeConsumerGroup(groupId)
     assertEquals(1, consumerSummaries.size)
-    assertEquals(Set(tp, tp2), consumerSummaries.head.assignment.toSet)
+    assertEquals(Set(tp, tp2), consumerSummaries.get.head.assignment.toSet)
```

Comment:
Ah, you're right, we do have the dead case covered. I was puzzled because I was expecting an update in this patch, but I guess `isEmpty` works for `Option` as well. So the only uncovered case is a group which is in the middle of a rebalance. This is harder to test without mocking. The only thing I can think of is to send a JoinGroup request directly through `NetworkClient` and then immediately call describeGroup(). That seems like a lot of work for a small gain, so maybe it's fine without it.